astropy/wcs/wcs.py

```diff
@@ -2446,6 +2448,26 @@ def sip_foc2pix(self, *args):
         """.format(docstrings.TWO_OR_MORE_ARGS('2', 8),
                    docstrings.RETURNS('pixel coordinates', 8))
 
+    def proj_plane_pixel_scales(self):
+        """Same as :func:`astropy.wcs.utils.proj_plane_pixel_scales`
+        but returns a list of `~astropy.units.Quantity`.
+
+        """
```

Comment:
If this function is going to be the new "workhorse", and also because it is a method of the main class `WCS` from this sub-package, I think it would be better to simply copy the docstring from the `utils` function and maybe make a mention like "For unit-un-aware version - see ..." (something like that)?

Reply:
:+1: - and one can have a `See Also` section to refer the the unit unaware function.